utils/template_library/Resource.tmpl

```diff
@@ -0,0 +1,8 @@
+
+
+class {{ klass }}(Resource):
+    def __init__(self, {{ container }}):
+        super({{ klass }}, self).__init__({{ container }})
+        self._meta_data['template_generated'] = True
+        self._meta_data['required_json_kind'] = {{ kind }}
+        self._meta_data['attribute_registry'] = {{ attr_reg_dict }}
```

Comment:
This is getting generated as follows:
```
class Pool(Resource):
    def __init__(self, Pools):
        super(Pool, self).__init__(Pools)
        self._meta_data['template_generated'] = True
        self._meta_data['required_json_kind'] = tm:ltm:pool:poolstate
        self._meta_data['attribute_registry'] =
```

The attribute_registry should be a dict.